dev/source/docs/building-setup-windows10.rst

```diff
@@ -1,75 +1,61 @@
 
 .. _building-setup-windows10:
 
-===========================================================
-Setting up the waf Build Environment on Windows10 using WSL
-===========================================================
+================================================================
+Setting up the Build Environment on Windows10 using WSL1 or WSL2
+================================================================
 
 These setup instructions describe how to setup "Bash on Ubuntu on Windows" (aka "Windows Subsystem for Linux") which allows building with waf.
 
 .. image:: ../images/build-on-windows10-top-image.jpg
     :target: ../_images/build-on-windows10-top-image.jpg
 
-Setup steps
------------
+It involves enabling the built-in Windows Linux environment (WSL) in Windows10,installing a compatible Linux OS image, and finally installing the build environment as if it were a normal Linux system.
```

Comment:
Missed spaces in "...Windows 10, installing..."

Reply:
done